### ISSUE TITLE
Issue: Update latest logos to the social link Icons

### ISSUE THREAD

**Issue by sejal-Kamble18** [nishantharkut/W3nity#71]
### Description:
Currently, the logos for Discord, LinkedIn, and GitHub are static and not linked to their respective social profiles. To improve user experience and accessibility, these icons should be updated to function as clickable links that redirect users to the correct social platforms.

Expected Behavior :
The lastest version of logo requied in 
- Discord logo 
- LinkedIn logo
- GitHub logo 

### Currently  :

<img width="518" height="300" alt="Image" src="https://github.com/user-attachments/assets/4801a3b8-b955-47a8-a761-025fbd66173b" />
